packages/eds-core-react/src/components/Menu/MenuItem.tsx

```diff
@@ -61,9 +61,6 @@ const Item = styled.button.attrs<StyleAttrsProps>(({ $isFocused }) => ({
           svg {
             fill: ${icon.states.disabled.typography.color};
           }
-          &:focus {
```

Comment:
Will this get the default browser focus now?

Reply:
nothing happens on focus now, only when focusing using keyboard (`focus-visible`) where it gets the same moss-green dashed focus as our other components.